src/main/java/com/minecolonies/core/commands/generalcommands/CommandGetRanks.java

```diff
@@ -0,0 +1,142 @@
+package com.minecolonies.core.commands.generalcommands;
+
+import com.minecolonies.api.colony.IColony;
+import com.minecolonies.api.colony.IColonyManager;
+import com.minecolonies.api.colony.permissions.Rank;
+import com.minecolonies.api.util.constant.translation.CommandTranslationConstants;
+import com.minecolonies.core.commands.commandTypes.IMCCommand;
+import com.mojang.authlib.GameProfile;
+import com.mojang.brigadier.arguments.IntegerArgumentType;
+import com.mojang.brigadier.builder.LiteralArgumentBuilder;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import net.minecraft.ChatFormatting;
+import net.minecraft.commands.CommandSourceStack;
+import net.minecraft.commands.arguments.GameProfileArgument;
+import net.minecraft.network.chat.ClickEvent;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.chat.Style;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.minecolonies.core.commands.CommandArgumentNames.PLAYERNAME_ARG;
+
+public class CommandGetRanks implements IMCCommand
```

Comment:
Can you switch the `IMCComand` to `IMCOPCommand`, so that only OPs may execute this